rtCommon/fileWatcher.py

```diff
@@ -214,12 +214,11 @@ def on_created(self, event):
     def on_modified(self, event):
         self.q.put((event, time.time()))
 
-
```

Comment:
Add back the blank line, (i.e. revert whitespace change).
We aren't always consistent, but usually have 2 blank lines between classes, and one blank line between methods of classes.

Reply:
added back blank spaces